eng/pipelines/runtime.yml

```diff
@@ -233,130 +233,148 @@ jobs:
             eq(dependencies.checkout.outputs['SetPathVars_coreclr.containsChange'], true),
             eq(variables['isFullMatrix'], true)))
 
-#
-# Build Mono debug
-# Only when libraries or mono changed
+# Build the whole product using Mono runtime
+# Only when libraries, mono or installer are changed
 #
 - template: /eng/pipelines/common/platform-matrix.yml
   parameters:
-    jobTemplate: /eng/pipelines/mono/templates/build-job.yml
+    jobTemplate: /eng/pipelines/common/global-build-job.yml
+    buildConfig: ${{ variables.debugOnPrReleaseOnRolling }}
     runtimeFlavor: mono
-    buildConfig: debug
     platforms:
-    - Android_x64
     - Android_x86
-    - Android_arm
     - Android_arm64
     - tvOS_x64
-    - tvOS_arm64
-    - iOS_x64
-    - iOS_x86
-    - iOS_arm
     - iOS_arm64
-    - OSX_x64
-    - Linux_x64
-    - Linux_arm
-    - Linux_arm64
-    - Linux_musl_x64
+    - iOS_x86
     - Browser_wasm
-    # - Linux_musl_arm64
-    - Windows_NT_x64
-    # - Windows_NT_x86
-    # - Windows_NT_arm
-    # - Windows_NT_arm64
+    - Linux_arm
     jobParameters:
+      testGroup: innerloop
+      nameSuffix: All_Mono
+      buildArgs: -s mono+libs+installer -c $(_BuildConfig)
+                 /p:StripSymbols=true /p:SkipTests=true
```

Comment:
I can't see any usage of `StripSymbols` anywhere in the source. Not sure where it was coming from? We might want to remove it.

Reply:
I was following what the installer was passing on their builds for these targets, but it might no longer be needed, I believe we might have made it the default behavior. Will remove.